Volume/Supporting/Secrets.swift

```diff
@@ -21,10 +21,10 @@ struct Secrets {
     static let announcementsHost = Secrets.keyDict["announcements-host"] as! String
     static let announcementsPath = Secrets.keyDict["announcements-path"] as! String
     static let announcementsScheme = Secrets.keyDict["announcements-scheme"] as! String
-    
     static let appdevWebsite = Secrets.keyDict["appdev-website"] as! String
+    static let downloadLink = Secrets.keyDict["download-link"] as! String
```

Comment:
Would `downloadURL` be more consistent with how we name urls in the rest of the app?